packages/cli/src/lifecycles/graph.js

```diff
@@ -86,17 +93,19 @@ const createGraphFromPages = async (pagesDir, config) => {
                 * relativeExpectedPath: relative import path for generated component within a list.js file to later be
                 * imported into app.js root component
                 * title: the head <title></title> text
+                * menu: the name of the menu in which this item can be listed and queried
```

Comment:
I started working on #274 which I think will be a better API for getting this kind of data from frontmatter into the graph instead.

Reply:
As #274 was merged to release/0.5.0 (this base branch), I've reimplemented the variables: menu, index, and linkheadings, tableOfContents, as part of the data object.